src/utils/openvpn.ts

```diff
@@ -362,7 +360,7 @@ export class VpnManager extends EventEmitter {
 				}
 				return this.waitForStart(since);
 			})
-			.return(true);
+			.return();
```

Comment:
`await` and ditch this
```suggestion
```

Reply:
This seems to be required otherwise we have a `Promise<unknown>` return value.

Reply:
Can you try again? I've got a feeling it was due to the `return` instead of the `await` in the previous version